constants.go

```diff
@@ -46,6 +46,11 @@ var (
 // UndefAddressString is the string used to represent an empty address when encoded to a string.
 var UndefAddressString = "<empty>"
 
+// MaxUint64StringLength defines the maximum length of `uint64` as a string.
+// FIXME: Can we dynamically get this from something like `math.MaxUint64.Bytes()`
+//  or is this guaranteed to be always the same value?
+const MaxUint64StringLength = 20
```

Comment:
Guaranteed to always be 20, since `ceil(log_10 2^64) = 20`, and therefore 2^64 is a 20-digit number.

(the "dynamic" way to get it would be to take `log_10` of `math.MaxUint64` and add 1)

Reply:
Or: `ceil(64.0*ln(2)/ln(10))`

Reply:
I agree on the numbers, this was not so much a mathematical question but about the consistency of the underlying representation to always be the same.